scripts/validate-brazil-config

```diff
@@ -12,7 +12,7 @@ import re
 # Usage: validate-brazil-config [module-paths-file] [dependencies-file]
 
 # Generating module-paths-file:
-#   mvn exec:exec -Dexec.executable=pwd -pl \!:aws-sdk-java-pom,\!:sdk-benchmarks,\!:module-path-tests -q 2>&1 > modules.txt
+#   mvn exec:exec -Dexec.executable=pwd -pl \!:aws-sdk-java-pom,\!:sdk-benchmarks,\!:http-client-benchmarks,\!:module-path-tests -q 2>&1 > modules.txt
```

Comment:
Can you update the buildspec as well?

https://github.com/aws/aws-sdk-java-v2/blob/cea37daa82e538612db3b4e1b99f29e1f332da6b/buildspecs/validate-brazil-config.yml#L12